vboard-front/src/main/common/vboardPin/templates/vboardPin.html

```diff
@@ -71,8 +71,8 @@
             <vboard-comment data-ng-if="createAllComments" ng-repeat="comment in comments | orderBy : 'post_date_utc' track by comment.id"></vboard-comment>
         </div>
         <div class="vboardPin--addComment" data-ng-if="connected && newComment">
-            <textarea placeholder="Ajouter un commentaire..." rows="5" maxlength="255" required></textarea>
-            <button type="button" data-ng-click="submitComment($event.target.value)"
```

Comment:
Just to understand : `$event.target.value` does not work here ?
Because it use at least in one other place in the code.
Also, what's the meaning of `$parent` in the template ?

Reply:
I don't know enough about Angular 1 to answer properly, but I don't see how the click event on a button would somehow pick up the textarea value.
As I understand it, `$parent` is an alias for the parent scope; since some `data-ng-*` attributes create scopes for their child (`data-ng-if` in that case), `$parent` refers to the component scope.
I think another way to do that would be, as indicated in the link you shared below, to have a nested object in the component scope (something like `$scope.comment.text` for instance).

Reply:
Sorry, had not seen your comment! No, it should be two lines of code tops. I'll look into it!